src/main/flight/pid.c

```diff
@@ -294,9 +294,8 @@ float pidCompensateThrustLinearization(float throttle)
 {
     if (pidRuntime.thrustLinearization != 0.0f) {
         // for whoops where a lot of TL is needed, allow more throttle boost
-        const float throttleCompensateAmount = (1.0f - 0.5f * pidRuntime.thrustLinearization);
         const float throttleReversed = (1.0f - throttle);
-        throttle /= 1.0f + throttleCompensateAmount * powerf(throttleReversed, 2) * pidRuntime.thrustLinearization;
+        throttle /= 1.0f + pidRuntime.throttleCompensateAmount * powerf(throttleReversed, 2) * pidRuntime.thrustLinearization;
```

Comment:
In this case you could even go so far as to initialise `pidRuntime.throttleCompensateAmount` as `(1.0f - 0.5f * pidRuntime.thrustLinearization) * pidRuntime.thrustLinearization` (or `pidRuntime.thrustLinearization - 0.5f * powerf(pidRuntime.thrustLinearization, 2)` and then calculate `throttle /= 1.0f + pidRuntime.throttleCompensateAmount * powerf(throttleReversed, 2)` here.

Reply:
Yes! :-)